### PR TITLE
#61 summon, milled, repo flags

### PR DESCRIPTION
**Pull request by valentinedwv**
Add some flags to improve usability

### CODE REVIEW

File: earthcube_utilities/src/ec/missing_report.py [hunk 1]
```diff
@@ -42,7 +46,6 @@ def writeMissingReport(args):
                 s3Minio.putReportFile(bucket, repo, "missing_report.json", report)
         except Exception as e:
             logging.error(f"could not write missing report for {repo} to s3server:{s3server}:{bucket} error:{e}", repo,s3server,bucket, e)
-            return 1
```
Comment (ylyangtw) [inline earthcube_utilities/src/ec/missing_report.py:45]: Just commited one change here. This way the program will still continue even one repository fails at writing missing report

File: earthcube_utilities/src/ec/missing_report.py [hunk 2]
```diff
@@ -27,13 +28,16 @@ def writeMissingReport(args):
     s3Minio = s3.MinioDatastore(s3server, None)
     sources = getSitemapSourcesFromGleaner(args.cfgfile)
     sources = list(filter(lambda source: source.get('active'), sources))
+    repos=args.source
```
Comment (ylyangtw) [inline earthcube_utilities/src/ec/missing_report.py:31]: I am checking this args.source but have a quick question. What should the input look like? Seems like for an added repository, both name and url are required

Comment (valentinedwv) [reply earthcube_utilities/src/ec/missing_report.py:31]: sources is a list from the gleaner...  aka this was used to load the data. The --source is just
The passed CLI arguement--source is the  {soruce.name} which is matched/filtered in line 35, below is where the action is.
if it finds the name of the repo in the passed parameters (sources) then it runs.

refactorig to use the same terminology

Comment (ylyangtw) [reply earthcube_utilities/src/ec/missing_report.py]: I understand now. Thanks for explaining :)

**Pull request merged**